M2/Macaulay2/d/stdiop.d

```diff
@@ -118,7 +118,7 @@ export tostring(w:Position) : string := (
 	     if c == ' ' then (
 		 filename = "\"" + filename + "\"";
 		 break));
-	 errfmt(filename ,int(w.line),int(w.column),int(w.loadDepth))));
+	 errfmt(filename, int(w.lineL), int(w.columnL), int(w.loadDepth))));
```

Comment:
Should this point to `lineF` and `columnF` (the focused location) instead, or just the beginning?

Reply:
I'm confused. the current `line`, `column`, *are* the focus position.

Reply:
Sure, that was your choice, but it's misleading given the top-level format for the _same object_ and the existing documentation, so we move the focus location to the end instead, and make the corresponding changes everywhere else a position is constructed. The names `line1`, `line2`, etc. shouldn't have been used in the first place, especially given that there were no commentary provided and the documentation of the object online pointed to the last pair as the focus position.

Reply:
if you change the ordering in the definition of `export Position`, don't you have to change every line of code that defines a new Position to take into account that new ordering? I don't see those changes.

Reply:
In all the other ones all three locations are the same. I changed all the ones where they're different. Have I missed any?

Reply:
OK I missed the unexpanded changes in `convertr.d`. looks good.

Reply:
anyway back to the start of this thread, I do believe it should be `lineF` and `columnF`, yes.